### PR TITLE
Feat: update contributing file

### PR DESCRIPTION
**Pull request by AlexTheWizardL**
# Naming Patterns

The Naming Patterns section establishes a consistent and structured approach for naming branches, commits, and pull requests in a repository. This consistency helps developers understand the purpose of each branch, commit, or pull request at a glance. Below is a description to include in your pull request.

## Naming Patterns Overview
To maintain consistency and clarity within the repository, we follow specific naming patterns for branches, commits, and pull requests. These patterns make it easier to track changes and identify the purpose of each item.

### Branch name
```
(feat|fix|docs|style|refactor|test|revert)/taskId_task-short-description
Ex.: feat/1_add-commit-and-branch-name-styles-to-readme
```

### Commit name
```
(feat|fix|docs|style|refactor|test|revert): update descriptions
Ex.: bug: update readme
```

### Commit description
```
tickets: #taskId, #taskId
Ex.: tickets: #1, #2, #3
```

### Pull-request name
```
(Feat|Fix|Docs|Style|Refactor|Test|Revert): #taskId description
Ex.: Fix: #15 unnecessary request removed
```

## Installing Pytest for Testing

To run tests with pytest, ensure you have it installed on your system. Pytest is a widely used testing framework in the Python ecosystem, known for its ease of use and support for various types of tests, including unit and integration tests.

To install pytest, follow these steps:

1. Open a terminal or command-line interface.
2. Run the following command to install pytest:

```bash
pip install pytest
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,6 +86,32 @@ Follow these steps to set up your local development environment on Unix/Mac OS:
...
+### Commit description
+```
+tickets: taskId, taskId
+Ex.: tickets: 1, 2, 3
+```
+
+### Pull-request name
+```
+(Feat|Fix|Docs|Style|Refactor|Test|Revert)/taskId description
+Ex.: Fix/15 unnecessary request removed
```
Comment (lasuk) [inline CONTRIBUTING.md:112]: Again: follow GitHub conventions for issue referencing, e.g. `#15`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -96,6 +122,11 @@ executed  through GitHub Actions after each commit, during pull requests,
 and daily. The tests connect to a private CashCtrl test account with the API
 key stored as a GitHub secret.
 
+Before you try to run tests, execute the following command, to install pytest
```
Comment (lasuk) [inline CONTRIBUTING.md:125]: line 125 can be shortened: ~~Before you try to run tests,~~ Execute the following command to install pytest:

**Review by lasuk — COMMENTED** [review]
Thanks a lot for improving our guidelines.

I left a few minor comments. Please implement or discuss further.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -86,6 +86,32 @@ Follow these steps to set up your local development environment on Unix/Mac OS:
...
+### Commit name
+```
+(feat|fix|docs|style|refactor|test|revert): update descriptions
+Ex.: bug: update readme
+```
+
+### Commit description
+```
+tickets: taskId, taskId
+Ex.: tickets: 1, 2, 3
```
[the remark this answers is not in the document]
Comment (AlexTheWizardL) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 1 above)
Comment (AlexTheWizardL) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (AlexTheWizardL) [reply CONTRIBUTING.md]: Fixed

**Comment by lasuk** [comment]
Merged. Thanks a lot!

**Pull request merged**